app/code/extensions/FooterConfig.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace App\extensions;
+
+use SilverStripe\Assets\Image;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\TextField;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\Forms\NumericField;
+use SilverStripe\AssetAdmin\Forms\UploadField;
+use SilverStripe\Forms\HTMLEditor\HTMLEditorField;
+
+class FooterConfig extends DataExtension
+{
+
+    private static $db = [
+        'PhoneNumber' => 'Varchar',
+        'Email' => 'Varchar',
+        'Facebook' => 'Varchar',
+        'FooterSentence1' => 'Varchar',
+        'FooterSentence2' => 'Varchar',
+    ];
+
+    private static $has_one = [
+        'FooterLogo' => Image::class,
+        'SponsorLogo1' => Image::class,
+        'SponsorLogo2' => Image::class,
+        'SponsorLogo3' => Image::class,
+        'SponsorLogo4' => Image::class,
+        'SponsorLogo5' => Image::class,
+        'SponsorLogo6' => Image::class,
+    ];
+
+    private static $owns = [
+        'FooterLogo',
+        'SponsorLogo1',
+        'SponsorLogo2',
+        'SponsorLogo3',
+        'SponsorLogo4',
+        'SponsorLogo5',
+        'SponsorLogo6'
+        ];
+
+
+    /**
+     * Create fields in the settings pane of the CMS
+     *
+     * @param FieldList $fields
+     * @return void
+     */
+    public function updateCMSFields(FieldList $fields)
+    {
+        $fields->addFieldToTab('Root.Main', TextField::create(
+            'PhoneNumber',
+
```

Comment:
There appears to be some additional spacing here as well as for the others below